src/pages/RepoPage/CoverageTab/CoverageTab.spec.tsx

```diff
@@ -4,12 +4,14 @@ import { graphql, rest } from 'msw'
 import { setupServer } from 'msw/node'
 import { MemoryRouter, Route } from 'react-router-dom'
 
-import { TierNames } from 'services/tier'
+import { TierNames, TTierNames } from 'services/tier'
 import { useFlags } from 'shared/featureFlags'
 
 import CoverageTab from './CoverageTab'
 
 jest.mock('shared/featureFlags')
+const mockedUseFlags = useFlags as jest.Mock<{ multipleTiers: boolean }>
```

Comment:
We **really** need to get rid of this flag, the team plan has been out for long enough